include/drivers/display/ili9340.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright (c) 2017 dXplore
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef DISPLAY_ILI9340_H
+#define DISPLAY_ILI9340_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <device.h>
+
+/**
+ * @brief Initialize ILI9340 display driver
+ *
+ * @param dev Pointer to device structure
+ *
+ * @return 0 in case of no error
+ */
+int ili9340_init(struct device *dev);
```

Comment:
do not expose device driver initialization function. DEVICE_INIT is using it properly already.